app/src/main/java/org/mozilla/fenix/settings/account/AccountSettingsFragment.kt

```diff
@@ -329,7 +332,10 @@ class AccountSettingsFragment : PreferenceFragmentCompat() {
         viewLifecycleOwner.lifecycleScope.launch {
             SyncAccount.syncNow.record(NoExtras())
             // Trigger a sync.
-            requireComponents.backgroundServices.accountManager.syncNow(SyncReason.User)
+            requireComponents.backgroundServices.accountManager.syncNow(
+                reason = SyncReason.User,
+                debounce = true,
```

Comment:
We need a full sync here on the account page - the user is explicitly asking to sync _now_.

Reply:
I'll remove.  Thanks